profiles/SampleMoonMage-setup.yaml

```diff
@@ -0,0 +1,129 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Sorcery (Does not include Lunar Magic)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# You will need a runestone in your inventory to train sorcery and arcana. Grab one from any donation shelf.
+# You will need to add more magic and survival skills depending on your circle
+# However, you will probably end up training them all since they each have very useful spells
+# The other Magic skills are: Targeted Magic, Warding, Debilitation, Utility
+
+# Hunting settings
+training_manager_hunting_priority: true
```

Comment:
I kind of like having the combats in there personally, eventually you need to train TM and/or Debil anyway to get 7 magics. I think having the weapons and shield would be pretty normal, its similar to what i do on Crann. Attunement gets locked after about three uses of perc mana, I think tactics took the longest.